plugins/smart_proxy_ansible/control

```diff
@@ -2,7 +2,7 @@ Source: ruby-smart-proxy-ansible
 Section: ruby
 Priority: optional
 Maintainer: Ivan Nečas <inecas@redhat.com>
-Build-Depends: debhelper, gem2deb
+Build-Depends: debhelper, gem2deb (>= 0.38.0~)
```

Comment:
@mmoll I've updated it here, https://tracker.debian.org/pkg/gem2deb indicates it should be available in unstable now. Does this sound sensible or am I missing something?